src/components/guidance-content/info/InformationItems.js

```diff
@@ -0,0 +1,617 @@
+// Central resource for LabelWithInfo and Information items used throughout
+// the app.
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Glyphicon, Table } from 'react-bootstrap';
+import LabelWithInfo from '../../guidance-tools/LabelWithInfo';
+import Information from '../../guidance-tools/Information';
+import css from './InformationItems.css';
+
+///////////////////////////////
+// Parts for building items
+///////////////////////////////
+
+
+///////////////////////////////
+// Contact info
+///////////////////////////////
+
+export const appContact = {
+  email: 'mailto:rglover@uvic.ca',
+  name: 'Rod Glover',
+};
+
+
+///////////////////////////////
+// Dataset filters (Model, Emission, Variable)
+///////////////////////////////
+
+export const datasetFilterPanelLabel = (
+  <LabelWithInfo label='Dataset Filter'>
+    <p>
+      This is the primary filter for the datasets to be displayed.
+    </p>
+    <p>
+      Datasets matching these filter criteria can be displayed in the
+      map, on graphs, and in a statistical summary table.
+      Within each such presentation of data, you can independently select
+      which of the matching datasets to display.
+    </p>
+  </LabelWithInfo>
+);
+
+export const modelSelectorLabel = (
+  <LabelWithInfo label='Model'>
+    <p>
+      GCM model with which the climate data was generated.
+    </p>
+    <p>
+      Models are identified by short codes. For full model identification,
```

Comment:
Maybe add a single sentence about what a model _is_, like "A model is a thermodynamics simulation of climate processes in the atmosphere and on earth's surface" or something?